2010-08-22  Samuel Thibault  <samuel.thibault@ens-lyon.org>

        * sysdeps/mach/i386/sysdep.h [__ASSEMBLER__] (__i686): New macro.

---
 sysdep.h |    5 +++++
 1 file changed, 5 insertions(+)

--- a/sysdeps/mach/i386/sysdep.h.orig	2010-08-22 16:44:46.000000000 +0200
+++ b/sysdeps/mach/i386/sysdep.h	2010-08-22 16:45:35.000000000 +0200
@@ -22,6 +22,11 @@
 #include <dl-sysdep.h>
 #include <tls.h>
 
+#if defined __i686 && defined __ASSEMBLER__
+#undef __i686
+#define __i686 __i686
+#endif
+
 #define LOSE asm volatile ("hlt")
 
 #define SNARF_ARGS(entry_sp, argc, argv, envp)				      \
